nemo/collections/asr/parts/utils/nmesc_clustering.py

```diff
@@ -1117,8 +1109,15 @@ def forward(
         embeddings_in_scales: torch.Tensor,
         timestamps_in_scales: torch.Tensor,
         multiscale_segment_counts: torch.LongTensor,
+        multiscale_weights: torch.Tensor,
         oracle_num_speakers: torch.LongTensor,
-    ):
+        max_num_speakers: int = 8,
```

Comment:
These need to be torch tensors (like other parameters)

Reply:
fixed